python/mxnet/libinfo.py

```diff
@@ -76,6 +76,34 @@ def find_lib_path():
         os.environ['PATH'] = os.environ['PATH'] + ';' + os.path.dirname(lib_path[0])
     return lib_path
 
+def find_include_path():
+    """Find MXNet included header files.
+
+     Returns
+    -------
+    incl_path : string
```

Comment:
as we discussed, return a list of string

Reply:
Keep the return type to string as we discussed offline